gossip/gossip_test.go

```diff
@@ -37,7 +36,7 @@ import (
 func TestGossipInfoStore(t *testing.T) {
 	defer leaktest.AfterTest(t)
 	rpcContext := rpc.NewContext(&base.Context{}, hlc.NewClock(hlc.UnixNano), nil)
-	g := New(rpcContext, TestBootstrap)
+	g := New(rpcContext, TestBootstrap, stop.NewStopper())
```

Comment:
We shouldn't create a `Stopper` without stopping it. I know it's not used here because we don't call anything that needs a stopper, but I think it would be better to either pass `nil` or assign the stopper to a local variable and stop it.